src/language-helper.js

```diff
@@ -0,0 +1,113 @@
+const counterpart = require('counterpart');
+
+const DEFAULT_LOCALE = "en";
+
+function _td(text) {
+    return _t(text);
```

Comment:
what's the point of having `_td` if it is just the same as `_t`?
In Web&React SDK `_td` is *mark as translatable* for the i18n-gen script, but does not do anything and just returns the string. Using `_td` everywhere where the other two layers would actually use `_t` feels backwards.

Reply:
Makes sense, my understanding of `_t` and `_td` was wrong. Will update